scripts/uberenv/spack_configs/toss_3_x86_64_ib/packages.yaml

```diff
@@ -26,9 +26,9 @@ packages:
     buildable: false
   mvapich2:
     externals:
-    - spec: mvapich2@2.3%clang@10.0.0 arch=linux-rhel7-ivybridge
+    - spec: mvapich2@2.3%clang@10.0.0 process_managers=slurm arch=linux-rhel7-ivybridge
```

Comment:
This missing causes very subtle problems if you are trying to find the mpi wrapper via the process manager which was suggested by Greg for compatibility issues.  Axom does this and we should eventually.

Reply:
yes, I understand some of those words.... 😄 I trust you on this one.